specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml

```diff
@@ -5,7 +5,7 @@ cite:
       doi: TODO
 authors:
     - Constantin Pape
-documentation: normalize_zero_mean_unit_varaince.md
+documentation: ./NormalizeZeroMeanUnitVariance.md
```

Comment:
Have we decided on any fixed schema to refer to local files?
Should it always be `./SomeFile`? Because this is missing above (E.g. in the BCELoss), where it's only `BCELoss.md`.

Reply:
@FynnBe Just let me know what's the current convention, I can change this myself then, if necessary.

Reply:
I oriented myself on the way `pathlib.Path` interpretes strings. Anything not starting with a slash or drive letter would be considered a relative path (`example.md == ./example.md`). e.g. the `./` change was unnecessary, however, I did change the file name (stem) to be the same as the yml itself... (which also does not matter, I simply found it more consistent)